pkg/controllers/management/kontainerdrivermetadata/data.go

```diff
@@ -314,7 +334,7 @@ func getLabelMap(k8sVersion string, data map[string]map[string]string,
 	if err != nil {
 		return nil, fmt.Errorf("k8sVersion not sem-ver %s %v", k8sVersion, err)
 	}
-	labelMap := map[string]string{}
+	labelMap := map[string]string{"cattle.io/creator": "norman"}
```

Comment:
Why do you need to do this?

Reply:
our crds always get this label, so if I don't consider it for comparison -- there's always need to update labels on sync